src/System.Private.CoreLib/shared/System/Text/StringBuilder.cs

```diff
@@ -1150,6 +1150,7 @@ public StringBuilder Remove(int startIndex, int length)
 
         public StringBuilder Append(bool value) => Append(value.ToString());
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
This is used in a ton of places. Why is the associated size increase acceptable / worthwhile?

Reply:
btw, are there any plans for a "size diff" CI job? (e.g. run jit-diff on demand for CoreLib, BCL, some other projects and report size changes)

Reply:
It is somewhat tricky to use jit-diff for corelib changes, as it requires multiple builds and multiple steps. But something like this would be useful.